spec/javascripts/details/views/notification_bar_spec.js.coffee

```diff
@@ -8,35 +9,138 @@
 describe 'Notification', ->
 
   describe '#setup', ->
-    it "set in view mode", ->
+    
+    it "sets in view mode", ->
       notification = new Notifications(getNotificationContent())  
       expect(notification instanceof Notifications).to.eql(true)
   
   describe '#_postNotification', ->
-    it "create a notification", ->
+    
+    it "creates a notification without type", ->
+      notification = new Notifications(getNotificationContent())  
+      expect(notification.el.find('ul li').length).to.eql(0)
+      notice = notification._prepare({type: "", content: "This is content", title: ""})
+      notification._postNotification(notice)
+      expect(notification.el.find('ul li').length).to.eql(1)
+      expect(notification.el.find('ul li.notice-type').length).to.eql(0)
+      expect(notification.el.find('ul li.notice-type-primary').length).to.eql(1)
+      expect(notification.el.find('ul li .notification-content').text()).to.eql("This is content")
+    
+    it "creates a notification with type", ->
+      notification = new Notifications(getNotificationContent())  
+      expect(notification.el.find('ul li').length).to.eql(0)
+      notice = notification._prepare({type: "success", content: "This is content", title: ""})
+      notification._postNotification(notice)
+      expect(notification.el.find('ul li').length).to.eql(1)
+      expect(notification.el.find('ul li.notice-type-success').length).to.eql(1)
+      expect(notification.el.find('ul li.notice-type-primary').length).to.eql(0)
+    
+    it "creates a notification with title", ->
+      notification = new Notifications(getNotificationContent())  
+      expect(notification.el.find('ul li').length).to.eql(0)
+      notice = notification._prepare({type: "success", content: "This is content", title: "This is a title"})
+      notification._postNotification(notice)
+      expect(notification.el.find('ul li .notification-title').text()).to.eql("This is a title")
+
+  describe '#_prepareOnLoadNotifications', ->
+    
+    it "creates a notification for each meta tag on page load", ->
```

Comment:
OK, this looks better now, but the `each` in the description implies it should be testing with more than one item. Maybe add two `meta` tags to `el` and check length of `2`?